packages/wrap/src/transforms/TransformInputObjectFields.ts

```diff
@@ -165,7 +167,10 @@ export default class TransformInputObjectFields<TContext = Record<string, any>>
             // The casting is kind of legit here as we are in a visitor
             const parentType = typeInfo.getInputType() as Maybe<GraphQLInputObjectType>;
             if (parentType != null) {
-              const parentTypeName = parentType.name;
+              const parentTypeName =
+                isWrappingType(parentType) && isInputObjectType(parentType.ofType)
```

Comment:
Related https://github.com/ardatan/graphql-tools/pull/4676/files#r956002686
Can we do `getNamedType` here because `ofType` can be also another wrapped type, right?

Reply:
Well spotted! you can indeed have an "infinite" amount of wraps, will change it

Reply:
Done!

Reply:
Awesome! Then could you create a changeset using `yarn changeset` then we can merge and cut a release?

Reply:
Done! Please check if the changeset makes sense 👍